quiz/quiz/quiz.py

```diff
@@ -0,0 +1,170 @@
+"""Welcome to Reflex! This file outlines the steps to create a basic app."""
+import reflex as rx
+import copy
+from .results import results
+from typing import Any
+from typing import List
+
+question_style = {
+    "bg": "white",
+    "padding": "2em",
+    "border_radius": "25px",
+    "width": "100%",
+    "align_items": "start",
+}
+
+
+class State(rx.State):
+    """The app state."""
+
+    default_answers = [None, None, [False, False, False, False, False]]
+    answers: List[Any]
+    answer_key = ["False", "[10, 20, 30, 40]", [False, False, True, True, True]]
+    score: int
+
+    def onload(self):
+        self.answers = copy.deepcopy(self.default_answers)
+
+    def set_answers(self, answer, index, sub_index=None):
+        if sub_index is None:
+            self.answers[index] = answer
+        else:
+            self.answers[index][sub_index] = answer
+
+    def submit(self):
+        total, correct = 0, 0
+        for i in range(len(self.answers)):
+            if self.answers[i] == self.answer_key[i]:
+                correct += 1
+            total += 1
+        self.score = int(correct / total * 100)
+        return rx.redirect("/result")
+
+    @rx.var
+    def percent_score(self):
+        return f"{self.score}%"
+
+
+def header():
+    return rx.vstack(
+        rx.heading("Python Quiz"),
+        rx.radix.separator(),
+        rx.text("Here is an example of a quiz made in Reflex."),
+        rx.text("Once submitted the results will be shown in the results page."),
+        style=question_style,
+    )
+
+
+def question1():
+    """The main view."""
+    return rx.vstack(
+        rx.heading("Question #1"),
+        rx.text(
+            "In Python 3, the maximum value for an integer is 26",
+            rx.chakra.text("3", as_="sup"),
```

Comment:
still has chakra text here

Reply:
Raised this PR(https://github.com/reflex-dev/reflex/pull/2620/files) to fix the issue with radix text component as `as_="sup"` wasnt supported.